pages/blog/20190921-builder.md

```diff
@@ -19,7 +19,7 @@ Why not have `terraform` already installed in the image?
 
 You can do that by adding a script to builder.
 
-
+**NOTE:** A Linux-based OS (Raspberry Pi works) is needed to create a new script for the builder repo properly.
```

Comment:
@law-lin this is the result but not the reason

-> script files need to be executable